<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>birthday_conv_ui</class>
 <widget class="QWidget" name="birthday_conv_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
    background-color: black;
	color: lime;
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: lime;
}

QPushButton:pressed{
   background-color : green;
}</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="enterButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
    background-color: black;
	color: lime;
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: lime;
}

QPushButton:pressed{
   background-color : green;
}</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Output">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: lime;
 border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: green;
    padding: 2px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgb(0, 0, 0);
color: rgb(55, 255, 0);
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: rgb(98, 255, 0);
    font: 11pt &quot;Arial&quot;;
}

QComboBox QAbstractItemView{
    color: lime;
    select-color: green;
    select-background-color: black;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Years</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Days</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Month">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: lime;
 border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: green;
    padding: 2px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Day">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: lime;
 border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: green;
    padding: 2px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Year">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: lime;
 border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: green;
    padding: 2px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> font: 11pt &quot;Arial&quot;;
color: lime</string>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Day">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;
color:lime;</string>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Year">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;
color: lime;</string>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Month</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Day</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Year</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
